Creacion de libreria

FabianRG1990 opened the pull request.

FabianRG1990 committed:
feat: creacion de libreria bashboard

creacion de libreria bashboard y root

FabianRG1990 committed:
feat: creacion de libreria shared

cvandradg reviewed (changes requested):
Tenemos que cambiar el prefijo para que no sea lib si no su workspace.
Tambien necestio evidencia de todo corriendo bien el e2e, lint,tests, build y serve

lint 
![image](https://github.com/user-attachments/assets/75dad6fc-4524-40a9-872a-b8a07b7457d5)

FabianRG1990 committed:
feat: prefix

correccion de el prefix para hacer que los selecttores de los componetes de dashboard sean fabian-dev en ves de lib-

FabianRG1990 committed:
feat: cambio de nombre en selector

olvide poner el nombre corecto del selector de dashboar
